src/index.js

```diff
@@ -311,23 +277,6 @@ class DAPIClient {
    */
   getUserByName(username) { return this.makeRequestToRandomDAPINode('getUser', { username }); }
```

Comment:
Hm, maybe we want to rename this one to `getIdentityByName` or something like that too?

Reply:
We are going to remove unused methods in forthcoming PR